src/Tribe/Views/V2/Views/By_Day_View.php

```diff
@@ -486,15 +488,39 @@ protected function backfill_multiday_event_ids( array $events = [] ) {
 				continue;
 			}
 
+			$site_timezone     = Timezones::build_timezone_object();
+			$utc               = Timezones::build_timezone_object( 'UTC' );
+			$use_site_timezone = Timezones::is_mode( 'site' );
+
 			/** @var \DateTime $event_day */
 			foreach ( $event_period as $event_day ) {
 				$event_day_string = $event_day->format( Dates::DBDATEFORMAT );
+				$start            = tribe_beginning_of_day( $event_day->format( Dates::DBDATETIMEFORMAT ) );
+				$end              = tribe_end_of_day( $event_day->format( Dates::DBDATETIMEFORMAT ) );
 
 				if ( ! isset( $this->grid_days_cache[ $event_day_string ] ) ) {
 					continue;
 				}
 
-				if ( ! in_array( $event_id, $this->grid_days_cache[ $event_day_string ], true ) ) {
+				// If the timezone setting is set to "manual timezone for each event" then this is correct.
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: Line exceeds maximum limit of 100 characters; contains 107 characters

<sub>hash: 90420e36aee9471ab95fba31edf4c45c</sub>